_posts/2016-03-24-blablanatomy.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout:     post
 title:      "BlaBlaNatomy"
-tags:       [organization]
+tags:       [organisation]
```

Comment:
I would rather stick with the US spelling (with the 'z') and change the reference in `_config.yml` to match it.